tables/output_tables.Rmd

```diff
@@ -73,6 +74,66 @@ histology_df %>%
 
 ```
 
+## Table 2: Patients with hypermutant or ultra-hypermutant tumors
+```{r table 2}
+# Read in coding TMB file, merge with histologies data
+tmb_hist <- read_tsv(file.path(data_dir, "pbta-snv-consensus-mutation-tmb-coding.tsv")) %>%
+    # rename TSB to Kids_First_Biospecimen_ID
+  rename(Kids_First_Biospecimen_ID = Tumor_Sample_Barcode) %>%
+  select(Kids_First_Biospecimen_ID, tmb) %>%
+  left_join(histology_df) 
+
+# Gather all PT_IDs (poi = "patients of interest") for those with any hypermutant samples which are not cell lines
+poi <- tmb_hist %>%
+  # filter to the hypermutators
+  dplyr::filter(tmb >= 10) %>%
+  pull(Kids_First_Participant_ID)
+  
+# Gather all BS_IDs (soi = "samples of interest"), whether hypermutant or not, for pts with hypermutant samples
+  soi_df <- tmb_hist %>%
+    filter(Kids_First_Participant_ID %in% poi) %>%
+    # Looks like there is a hypermutant metastatic NBL sample we had not caught previously, but let's remove here since we focus on CNS anyway
+    # keep only WGS samples; remove metastases 
+    filter(experimental_strategy == "WGS" & short_histology != "Metastases") %>%
+    # add manual fields
```

Comment:
Can you add some comments for where this information comes from?

Reply:
can you check what I added and see if that helps?

Reply:
I'm confused, did I miss something?

Reply:
There is now a table that prints, showing histologies of the hypermutators and this one is removed to focus on CNS

Reply:
Ah, I see. I was referring to adding some comments in to contextualize the _manually_ filled in rows that come after `# add manual fields`, and link to where this information was obtained with the OpenPBTA.

Reply:
OHH. I see - I will add the comment, but this was manual review of pathology reports by me and confirmation with clinical folks that this is the latest their databases show.

Reply:
Updated in [4da7831](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1326/commits/4da78316ba80a68b73921de5d69cbe2dde742c1f), but also deleted the html file, as it is out of date. It is not rendering though - thoughts?

Reply:
It renders for me here. What error are you seeing?

Reply:
weird - 

```r
  html_notebook: default
```

did not work, so I switched to this and it did 🤷‍♀️ 

```r
  html_notebook:
    toc: TRUE
    toc_float: TRUE
```